akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -1,25 +1,35 @@
-# Cluster Membership
+# Cluster Usage
+  
+This document describes how to use Akka Cluster and the Cluster APIs. 
+For specific documentation topics see: 
+
+* @ref:[Cluster Specification](cluster-specification.md)
+* @ref:[Cluster Membership Service](cluster-membership.md)
+* @ref:[When and where to use Akka Cluster](choosing-cluster.md)
+* @ref:[Rolling Updates](../additional/rolling-updates.md)
+* @ref:[Operating, Managing, Observability](../additional/operations.md)
 
 ## Dependency
 
-To use Akka Cluster Typed, you must add the following dependency in your project:
+To use Akka Cluster add the following dependency in your project:
 
 @@dependency[sbt,Maven,Gradle] {
   group=com.typesafe.akka
   artifact=akka-cluster-typed_$scala.binary_version$
   version=$akka.version$
 }
 
-## Introduction
+## Cluster API Extension
 
-For an introduction to Akka Cluster concepts see @ref:[Cluster Specification](../common/cluster.md). This documentation shows how to use the typed
-Cluster API.
+The Cluster extension gives you access to management tasks such as @ref:[Joining, Leaving and Downing](cluster-membership.md#user-actions)
+and subscription of cluster membership events such as @ref:[MemberUp, MemberRemoved and UnreachableMember](cluster-membership.md#membership-lifecycle),
+which are exposed as event APIs.  
 
-You need to enable @ref:[serialization](../serialization.md) for your actor messages.
-@ref:[Serialization with Jackson](../serialization-jackson.md) is a good choice in many cases and our
-recommendation if you don't have other preference.
+It does this through these references are on the `Cluster` extension:
 
-## Examples
+* manager: An @scala[`ActorRef[ClusterCommand]`]@java[`ActorRef<ClusterCommand>`] where a `ClusterCommand` is a command such as: `Join`, `Leave` and `Down`
```

Comment:
The full package path to at least `ClusterCommand` (or api-doc-link)?

Reply:
Agreed. But I think maybe in a separate PR. If I start adding those here on this page then it should also be done for the rest of the page, etc. Since it didn't have that originally, for this PR I'd suggest https://github.com/akka/akka/issues/27739

Reply:
I don't think they need to be everywhere, but just mentioning that there are a few more `ClusterCommand` subtypes without mentioning where to find them seems bad.

Reply:
Good point, adding.